package.json

```diff
@@ -789,6 +795,10 @@
                 {
                     "command": "postgreSQL.connectDatabase",
                     "when": "view == azureDatabasesExplorer && viewItem == postgresDatabase"
+                },
+                {
+                    "command": "postgreSQL.createQuery",
```

Comment:
As a user, if my main goal is to create a function, I don't think I would know to right click at the database level and click "Create Query from template...". We don't have plans for a single other template (right?), so I'm not sure a generic "Create Query From Template..." command is the best idea at the moment.

I see this going one of two ways:
1. We figure out enough templates to make a generic "Create Query From Template..." command worth it
1. We refactor the wizard to be function-specific.

The first one sounds pretty complicated, so I prefer the second option. Why were we opposed to a simple "Create function..." command on the "Functions" node? We might have to mess with the messaging since the user is actually creating a query they have to execute, but overall that entry point feels much simpler and more discoverable

Reply:
If I remember right, we decided to go with the query feature so it could encompass stored procedures, tables, & anything else we might want to do via a query. And we wanted to avoid "Create Function..." just because it's really creating a query that would be executed to create a function. But I agree that creating a function isn't very discoverable under "Create Query From Template..." command.

I'd vote for refactoring the wizard to be function specific. For this release, we won't have any query options other than functions. And I would kinda like having separate commands for "Create Function Query From Template...", "Create Stored Procedure Query From Template...", etc. If that's something we want to do

Reply:
Also, "Create Function Query From Template..." sounds kinda cumbersome if we go with that option. What do you think of just "Create Function Query..." ?

Reply:
Well...there's two query features here:
1. Execute query
1. Create query template

IMO 1 still has to be generic enough to cover "anything else we might want to do via a query", but 2 is less important in that regard.

I'm fine with either name for the function query thing - easy to change anyways